### ISSUE TITLE
Tooltip: add `scale` property and `pointerDisabled` with the ability to disable the pointer

### ISSUE THREAD

**Issue by COV-GIS** [Esri/calcite-design-system#5334]
### Description

Tooltip with half the padding and possibly slightly smaller font size.

### Acceptance Criteria

Add boolean `compact` (or similar) property to accomplish the above.

Also...possibly a property to disable the arrow element. When combined with `compact`, toolitp would be more like a native browser title.

### Which Component

tooltip

### Example Use Case

For actions in bars or pads or in header slots the tooltip size is fine as is. But seems too big and overwhelming in content with lots of controls and such, especially when using small scale components.

A smaller tooltip without an arrow might also be able to replace native expand/collapse titles for blocks and accordions, and other similar situations. I don't know if that would create an accessibility issue or not. A mix of tooltips and native titles seems visually odd in my personal opinion.

# Calcite design solution
[Figma file](https://www.figma.com/file/MVmcY0ZyvVuPOnztxYLYhp/Tooltip---padding-and-scale?type=design&node-id=531-12657&mode=dev) available and includes:

- [x] Added scales
- [x] Reduced padding on medium scale when compared to current Tooltip
- [x] Added `pointer-disabled` property

![Image](https://github.com/user-attachments/assets/649bc904-6fa2-4246-aa5a-82d6b8100603)

**monday.com sync:** #9364726412

**Comment by macandcheese** [comment]
It does seem like we could offer scale property here to control the padding, and set a default font size. Currently you can add a custom declaration to adjust the font size of slotted content.

We expose `disable-pointer` on `calcite-popover`, that also seems like a reasonable addition to tooltip.

Thoughts cc @asangma @ashetland ?

**Comment by macandcheese** [comment]
Yes, `disable-pointer` shouldn't be limited to a single scale. I think using the `s`, `m`, `l` scale instead of default / compact makes sense. We use 10px on `s` `input-message`, so maybe worth evaluating that as well.

**Comment by ashetland** [comment]
Agree on a full scale set of `s`, `m`, `l`. Here's a Chrome tip with our 10px-font tooltip overlaid. Might be worth going to 11px to increase readability. 
<img width="467" alt="CleanShot 2022-09-22 at 14 22 31@2x" src="https://user-images.githubusercontent.com/108549080/191853914-e690b8bb-879f-494e-8ac4-213ce7090f06.png">

**Comment by jul11557** [comment]
+1 on this for ArcGIS Mission. We have no issues with the font size at 12, but would appreciate the option for smaller padding so that the tooltip doesn't block other UI elements.

**Comment by macandcheese** [comment]
To confirm from design, we plan to maintain the font size and reduce the default padding across the board, correct? @ashetland @SkyeSeitz

**Comment by ashetland** [comment]
Correct

**Comment by ashetland** [comment]
Updated designs posted [above](https://github.com/Esri/calcite-design-system/issues/5334#:~:text=Design%20recommendations).

**Comment by COV-GIS** [comment]
Is this ever going to be a thing?

If not, just let me know, so I can start working on my own component to meet my needs.

@geospatialem @isaacbraun @ashetland

**Comment by DitwanP** [comment]
@COV-GIS Apologies for the wait but yes, this is still planned and in development. It’s currently targeted to land in the 5.1 release in May.

Thanks for checking in!

**Comment by isaacbraun** [comment]
Verified the following on `5.1.0-next.21` via https://codepen.io/isaacbraun/pen/PwGEYGz:
- The `scale` property exists and functioning as expected, with the padding, font-size, and line-height adjusting as specified in Figma spec.
- The `pointer-disabled` property exists and disables the caret pointer as expected.
